etc/httpd/global.conf

```diff
@@ -126,10 +126,9 @@ RewriteRule ^/ja/?$ http://www.mozilla.jp/ [L,R]
 RewriteRule ^/ja/firefox/organizations/?$ http://www.mozilla.jp/business/downloads/ [L,R=301]
 
 # bug 1138280
-RewriteRule ^/ja/(firefox|thunderbird)/(beta/)?notes/ http://mozilla.jp/$1$2/notes/ [L,R=301]
-RewriteRule ^/ja/mobile/(beta/?)notes/ http://mozilla.jp/android/$1$2/notes/ [L,R=301]
-RewriteRule ^/ja/(firefox|thunderbird)/(\d+\.\d+(\.\d+)*)(beta)?/releasenotes(/?|/.+)$ http://mozilla.jp/firefox/$1/$2$3/releasenotes/ [L,R=301]
-RewriteRule ^/ja/mobile/(\d+\.\d+(\.\d+)*)(beta)?/releasenotes(/?|/.+)$ http://mozillla.jp/firefox/android/$1$2/releasenotes/ [L,R=301]
+RewriteRule ^/ja/(firefox|thunderbird)/(beta/)?notes/ http://www.mozilla.jp/$1$2/notes/ [L,R=301]
+RewriteRule ^/ja/(firefox|thunderbird)/((android/)?[0-9]+\.[0-9]+(\.[0-9]+)*(beta)?)/releasenotes/? http://www.mozilla.jp/$1/$2/releasenotes/ [L,R=301]
```

Comment:
I have updated my redirection rules with review comment.

    [0-9]+\.[0-9]+(\.[0-9]+)*(beta)?

This part may give us some confusion, but it is need to redirect the access to the URLs such as /ja/firefox/38.0.5beta/relasenotes/